release_notes/ocp_3_9_release_notes.adoc

```diff
@@ -0,0 +1,932 @@
+ConfigMap. Changes are auto-detected.
+
+[[ocp-39-notable-technical-changes]]
+== Notable Technical Changes
+
+{product-title} 3.9 introduces the following notable technical changes.
+
+[discrete]
+[[ansible-must-be-installed]]
+=== Ansible Must Be Installed via the rhel-7-server-ansible-2.4-rpms Channel
+
+Starting in {product-title} 3.9, Ansible must be installed via the
+`rhel-7-server-ansible-2.4-rpms` channel, which is included in RHEL
+subscriptions.
+
+[discrete]
+[[ocp-39-several-oc-secrets-subcommands-now-deprecated]]
+=== Several oc secrets Subcommands Now Deprecated
+
+{product-title} 3.9 deprecates the following `oc secrets` subcommands in favor
+of `oc create secret`:
+
+* `new`
+* `new-basicauth`
+* `new-dockercfg`
+* `new-sshauth`
+
+[discrete]
+[[updated-default-installer-values]]
+=== Updated Default Values for template_service_broker_prefix and template_service_broker_image_name in the Installer
+
+Default values for `template_service_broker_prefix` and
+`template_service_broker_image_name` in installer have been updated to be
+consistent with other settings.
+
+Previous values are:
+
+    * `template_service_broker_prefix="registry.example.com/"`
+    * `template_service_broker_image_name="origin-template-service-broker"`
+    * `template_service_broker_image_name="ose-template-service-broker"`
+
+New values are:
+
+    * `template_service_broker_prefix="registry.example.com/ose-"`
+    * `template_service_broker_prefix="registry.example.com/origin-"`
+    * `template_service_broker_image_name="template-service-broker"`
+
+[discrete]
+[[removed-become-no-instances]]
+=== Removed Several Instances of 'become: no' on Certain Tasks and Playbooks Inside of openshift-anisble
+
+In an effort to provide greater flexibility for users, several instances of
+`become: no` on certain tasks and playbooks inside of `openshift-anisble` are
+now removed. These statements were primarily applied on `local_action` and
+`delegate_to: localhost` commands for creating temporary files on the host
+running Ansible.
+
+If a user is running Ansible from a host that does not allow password-less
+`sudo`, some of these commands may fail if you run the `ansible-playbook` with
+the `-b` (`become`) command line switch, or if it has `ansible_become=True`
+applied to the local host in the inventory or `group_vars`.
+
+Elevated permissions are not required on the local host when running
+`openshift-ansible` plays.
+
+If target hosts (where {product-title} is being deployed) require the use of
+`become`, it is recommended that you add `ansible_become=True` for those hosts
+or groups in inventory or `group_vars`/`host_vars`.
+
+If the user is running as root on the local host or connection to the root user
+on the remote hosts instead of using become, then you should not notice a change.
+
+[discrete]
+[[unqualified-image-specs]]
+=== Unqualified Image Specifications
+
+Unqualified image specifications now default to `docker.io` and require API
```

Comment:
@frobware  help check this since card https://trello.com/c/w43PAXdU/846-5-node-image-policy-admission-plugin-for-qualifying-image-names is fixed with https://github.com/kubernetes/kubernetes/pull/58955